packages/checkout-ui-extensions/package.json

```diff
@@ -12,6 +12,9 @@
   "module": "index.mjs",
   "esnext": "index.esnext",
   "types": "./build/ts/index.d.ts",
+  "scripts": {
+    "build-docs": "sh ./docs/build-docs.sh"
+  },
```

Comment:
This file also needs an updated [generate-docs](https://github.com/Shopify/checkout-web/blob/master/packages/checkout-ui-extensions/package.json#L15-L17) dependency.